ext/opcache/zend_accelerator_module.c

```diff
@@ -924,6 +925,21 @@ ZEND_FUNCTION(opcache_invalidate)
 	}
 }
 
+/* {{{ Prevents JIT on function. Call it before the first invocation of the given function. */
+ZEND_FUNCTION(opcache_jit_blacklist)
```

Comment:
It would be good to have a least one test for this function.
I suppose, it can't be called at this moment... :)

There is a big question about usability of this API.
It can prevent and revert JIT-ing of already cached functions, but it can't disable setting JIT counters at the first place.

Reply:
It can be called, but not really tested for its functionality, unless you have a way how to check whether a function is actually executed under jit without looking at it in gdb.

Reply:
Oh, sorry, my mistake. I forgot about automatic registration through ``opcache.stub.php``.